teams/developer-relations/bible-for-developers/stage-4-breakdown.md

```diff
@@ -0,0 +1,115 @@
+# Stage 4 Breakdown
+
+#### Stage 4 - For Reviewers Only
+
+* [ ] **Bit Learning Management System \(LMS\)**
+
+  We have a LMS built to help us teach students step by step how to tackle complex and technical problems. In addition, we are open source so we are looking forward to finishing up LMS and creating small enough issues to let anyone contribute to our project.
+
+For the LMS to work, each module has a folder structure to follow.
+
+Walk through example:
+
+On the Bit curriculum Github, there is a module you should use as an example when trying to structure your module. Follow along by going to [https://bitproject.org/curriculum](https://bitproject.org/curriculum).
+
+Once at the link it should look like this. We’re going to go into the Data-Structures-and-Algos-Topic.
+
+![](https://lh6.googleusercontent.com/qvvB1JPKR_jm5GzbY8MyJtZ5XfJkzJiOsZTS6KswZLK_ivUf5REWuELLhDuImuoqxbSveiiCCtG8ok63KRArxcv28cCQC8ROfS2d2jkTB-9RlT7aCPOdmp95tXau3xAebj1o8VDy)
+
+You see how each module’s names are formatted Module\# name. Please do only use dashes \(-\) between words as any colons, semicolons, and slashes \(: ; / \) will mess with Github and cause trouble.
```

Comment:
This is worded weirdly. Maybe reword the second sentence better?

Reply:
I think there could be a smoother transition from mentioning the LMS to folder structure walkthrough.

Also, I would advise actually typing out the folder structure (outline format). It is more work, but for someone reading this, it will make much more sense.